crates/sui/multisig.md

```diff
@@ -0,0 +1,83 @@
+# Multisig
+
+Here we demonstrate the workflow to create a multisig transaction and submits to sui using CLI against Localnet. 
+
+## Step 1: Add some keys to keystore
+
+Here we use the sui.keystore to demonstrate. 
+
+```shell
+target/debug/sui client new-address secp256r1
```

Comment:
I'd use the canonical order ed, k1, r1 to avoid any accidental perceptional flag mismatches.